### ISSUE TITLE
having trouble running the game on windows 7

### ISSUE THREAD

**Issue by rschum** [Master-Foo/game#90]
is it a windows file path problem?

C:\Users\Richard\repositories\Master-Foo\game>C:\Python27\python.exe demo\demo.py
Traceback (most recent call last):
  File "demo\demo.py", line 6, in <module>   from source.systems.game import game
  File "C:\Users\Richard\repositories\Master-Foo\game\source\systems\game\game.py", line 2, in <module>  from model import model
  File "C:\Users\Richard\repositories\Master-Foo\game\source\systems\game\model\model.py", line 1, in <module> from source.abstract.app.model import model
  File "C:\Users\Richard\repositories\Master-Foo\game\source\abstract\app\model\model.py", line 1, in <module> from source.abstract.base_object.model import model
  File "C:\Users\Richard\repositories\Master-Foo\game\source\abstract\base_object\model\model.py", line 3, in <module> from source.library.science.math.geometry.g3d.sphere import sphere
  File "C:\Users\Richard\repositories\Master-Foo\game\source\library\science\math\geometry\g3d\sphere\sphere.py", line 1, in <module>  from ..primitive_3d import primitive_3d
  File "C:\Users\Richard\repositories\Master-Foo\game\source\library\science\math\geometry\g3d\primitive_3d\primitive_3d.py", line 1, in <module> from pygame import math 
ImportError: cannot import name math

C:\Users\Richard\repositories\Master-Foo\game>

**Comment by Master-Foo** [comment]
First, you need to be inside the game directory (which it looks like you are so that's probably not the problem, just an FYI)

Next, try the following:

```bash
.\run
```
or
```bash
python .\run
```
or
```bash
python .\demo\demo.py
```

**Comment by rschum** [comment]
C:\Users\Richard\repositories\game>.\run
'.\run' is not recognized as an internal or external command,
operable program or batch file.

C:\Users\Richard\repositories\game>python
'python' is not recognized as an internal or external command,
operable program or batch file.

so I have to specify the full path to python...
--------------
.C:\Users\Richard\repositories\game>C:\Python27\python.exe .\run
  File ".\run", line 5
    ./demo/demo.py
    ^
SyntaxError: invalid syntax

**Comment by rschum** [comment]
same problems when I translate to bash:

Richard@Richard-HP MINGW64 ~/repositories/game (master)
$ /c/python27/python.exe ./run
  File "./run", line 5
    ./demo/demo.py
    ^
SyntaxError: invalid syntax

**Comment by Master-Foo** [comment]
I'll leave the Windows nonsense as a activity you can figure out on your own. But IIRC when you install python in Windows it doesn't get put in your `$PATH`. So, if you add it to `$PATH` you should be able to run normally.

Also, IIRC, when you add something to your `$PATH`, you need to close out your teminal and re-open it.

Also, you may have to execute `.\demo\demo.py` `.\run` is a bash script and may not work on windows.

**Comment by Master-Foo** [comment]
In otherwords, reformat your harddrive to get rid of that virus named Windows and reinstall a REAL MAN's OS, AKA Linux.

**Comment by rschum** [comment]
thank you venerable master

**Comment by Master-Foo** [comment]
asldkfjasldkjsafjs

```python
asdf = 0
```

`asdf = 0`

**asdf**

__asdf__

#13 

# asdf

## asdf

### asdf

*asdf*

[asdf](asdf.com)

**Comment by rschum** [comment]
isn't this the crux of the problem?
while running the 32-bit  python 2.7 with 32-bit version of pygame installed:

>>> import pygame; print pygame.version.ver
1.9.1release
>>> from pygame import math
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name math
>>>

**Comment by Master-Foo** [comment]
How did you install `pygame`? Was it with `pip`?

When I run the same commands you listed everything works on my setup. The only difference being that my `pygame` version is `1.9.3` But there shouldn't be a major change between that and `1.9.1`. You **should** have `pygame.math` available if you have `pygame`.

TLDR; Fuck if I know.

Try this. Maybe it will work maybe it wont.

when you run `python` as a command in the terminal you **may** be using `Python 2`, you **may** be using `Python 3` It's hard to tell which is installed by default

So, do something like this to find out

```bash
> python
>>> import platform
>>> print platform.python_version()
2.7.12
```

it says you are using version `3.x.x` then you need to invoke `python2` or `python2.7` specifically when you run the game. So, something like this:

```bash
> python2 .\demo\demo.py
```

**Comment by rschum** [comment]
thanks, good learning this stuff.
\>\>\> import platform
\>\>\> print platform.python_version()
2.7.13

**Comment by rschum** [comment]
I found a 64 bit pygame version. I'm wondering if switching to everything to 64 bit might cause other problems?

**Comment by Master-Foo** [comment]
All I can say is try it.

**Comment by rschum** [comment]
i've posted on google groups to see if I can get help there. Haven't yet found any issues that are similar enough.

**Comment by rschum** [comment]
It appears I've got it working.  I believe the problem with pygame math is that you need 1.9.3, 1.9.1release won't work in windows. In your install guide you should get rid of the link 
pygame for py2.7 and pygame-sdl4 (32-bit)
    Windows Direct Download of pygame. 

this is what worked for me:
install python 2.7 (32bit)
\>pip install --upgrade pip
\>cd <cloned game directory>
\>pip install -r requirements.txt
\>demo\demo.py

**Comment by rschum** [comment]
I'll submit suggested changes for README.md on my fork.

**Comment by Master-Foo** [comment]
I'll show up at Jazzy around 14:00ish on Sunday. If you bring your laptop, I'll show you around the system so you can understand how everything works together.

**Comment by rschum** [comment]
My laptop is kaput. Will you show me around the system using your laptop?

**Comment by rschum** [comment]
suggested changes for the readme on game repo
https://github.com/rschum/game
